test/mocks/PLCRVotingMock.sol

```diff
@@ -0,0 +1,78 @@
+pragma solidity ^0.4.24;
+
+contract PLCRVotingMock {
+  address public token;
+  uint public pollNonce;
+  bool mock_isPassed;
+  bool mock_pollEnded;
+  uint mock_getTotalNumberOfTokensForWinningOption;
+  uint mock_getNumPassingTokens;
+
+  constructor(address _token) public {
+    token = _token;
+    pollNonce = 0;
+  }
+
+  // =========================
+  // startPoll Mock
+  // =========================
```

Comment:
could probably get away with removing the comments, I think the mock functions make sense without them